iotedgedev/envvars.py

```diff
@@ -215,13 +215,13 @@ def get_registries(self):
         for key in os.environ:
             key = key.lower()
             if key.startswith('container_registry_server'):
-                token = key[25:]
+                token = key[len('container_registry_server'):]
```

Comment:
I would abstract 'container_registry_server' into a const, so it is not repeated in code. If you need to change later you change in one place.